src/main/java/md/miller1995/Dealership/models/entities/CarEntity.java

```diff
@@ -13,6 +13,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class CarEntity {
+
     @Id
     @Column(name = "id")
     @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Use of simpler words maybe? Instead of date_of_manufacturesd maybe created_at?

Reply:
- renamed